### PR TITLE
V1 0 0

### PR DESCRIPTION
**Pull request by camagallen**
## Overview
Required updates to links and updates related to pypi release:
- Updated setup to use config (setup.cfg) to to align with pypi recommendations
- Cleaned dependencies and package validation (dropped unused modules)
- Updated tutorial validation as appropriate in line with the above changes
- fairness_metrics.py was restored to private status d.t. insufficient validation

### Note to Reviewers
- version.json has been dropped in favor of setup.cfg
- Notebooks were updated to remove library references that are unused. However, the __validation.validate_tutorial_requirements function was also updated, so notebooks were re-run to verify functionality. Because notebooks were re-run, the diffs will appear large. However, you only need to verify changes to the first two cells (other changes can be disregarded). Will add comment to the tops of the notebook Diffs as well, as a reminder.

## How was this update tested?
pytest & wheel build

## Checklist
- [x] I have read the [Contributing](https://github.com/KenSciResearch/docs/code_contributions/CONTRIBUTING.md) guidelines
- [x] I performed a self-review of my own code
- [x] The CHANGELOG and version.json have been updated
- [x] Documentation has been updated to reflect the changes
    - [x] The code has clear and helpful comments, particularly in hard-to-understand areas
    - [x] READMEs, Tutorials/Examples, and links have been updated as appropriate
- [x] Tests have been added to prove that the fix is effective or that the feature works
- [x] Pytest was run; new and existing unit tests pass locally with my changes
- [x] I have made note of any breakages or dependencies (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] These changes generate no new warnings

### CODE REVIEW

File: examples_and_tutorials/Example-ToolUsage_Regression.ipynb [hunk 1]
```diff
@@ -27,8 +27,6 @@
     "from fairmlhealth import report, measure, stat_utils\n",
     "import numpy as np\n",
     "import pandas as pd\n",
-    "import matplotlib.pyplot as plt\n",
```
Comment (camagallen) [inline examples_and_tutorials/Example-ToolUsage_Regression.ipynb:30]: @kenviku  This is the only actual change for this notebook. The seaborn and matplotlib libraries were being loaded but not used. Other changes are simply from re-running the existing notebook to validate **and can be ignored**.

File: examples_and_tutorials/Supplemental-ModelsForBinaryClassificationExample.ipynb [hunk 2]
```diff
@@ -64,19 +64,9 @@
     "from fairmlhealth import measure, stat_utils, report\n",
     "\n",
     "# Verify that additonal required packages are present (packages will be loaded later in the notebook)\n",
-    "from fairmlhealth.__validation import validate_notebook_requirements\n",
```
Comment (camagallen) [inline examples_and_tutorials/Supplemental-ModelsForBinaryClassificationExample.ipynb:67]: @kenviku  This is the only actual change for this notebook. Validation scripts were updated s.t. fairlearn installation is only enforced on the Supplemental notebook that actually uses it. Validation was also moved to ensure that it runs in the first cell. 

Other changes are simply from re-running the existing notebook to validate and **can be ignored**.

File: examples_and_tutorials/Tutorial-EvaluatingFairnessInBinaryClassification.ipynb [hunk 3]
```diff
@@ -104,6 +104,23 @@
   {
    "cell_type": "code",
    "execution_count": 1,
+   "metadata": {},
```
Comment (camagallen) [inline examples_and_tutorials/Tutorial-EvaluatingFairnessInBinaryClassification.ipynb:107]: @kenviku  This is the only actual change for this notebook. Validation was moved to ensure that it runs in the first cell. 

Other changes are simply from re-running the existing notebook to validate and **can be ignored**.

File: examples_and_tutorials/Tutorial-EvaluatingFairnessInRegression.ipynb [hunk 4]
```diff
@@ -63,6 +63,24 @@
   {
    "cell_type": "code",
    "execution_count": 1,
+   "metadata": {},
```
Comment (camagallen) [inline examples_and_tutorials/Tutorial-EvaluatingFairnessInRegression.ipynb:66]: @kenviku  This is the only actual change for this notebook. Validation was moved to ensure that it runs in the first cell. 

Other changes are simply from re-running the existing notebook to validate and **can be ignored**.

**Review by kenviku — APPROVED** [review]
LGTM

File: test/test_tutorial_evaluatingregression.py [hunk 5]
```diff
@@ -9,7 +9,6 @@
 from .notebook_tester import validate_notebook, list_warnings
 from .__utils import is_test_environment
 import os
-import sys
```
Comment (camagallen) [inline test/test_tutorial_evaluatingregression.py:12]: unused package

File: test/test_fairnessmetrics.py [hunk 6]
```diff
@@ -2,11 +2,10 @@
 Validation tests for fairmlhealth
 """
 
-import fairmlhealth.fairness_metrics as fm
+import fairmlhealth.__fairness_metrics as fm
 from numpy import isnan
 import pandas as pd
 import pytest
-from fairmlhealth import stat_utils
```
Comment (camagallen) [inline test/test_fairnessmetrics.py:9]: unused package

File: fairmlhealth/stat_utils.py [hunk 7]
```diff
@@ -7,7 +7,7 @@
 from scipy import stats
 from typing import Callable, Type
 from . import __preprocessing as prep, __validation as valid
-from .__validation import ArrayLike, IterableOfStrings, MatrixLike
```
Comment (camagallen) [inline fairmlhealth/stat_utils.py:10]: unused packages

File: fairmlhealth/performance_metrics.py [hunk 8]
```diff
@@ -1,8 +1,5 @@
 """
 """
-import copy
```
Comment (camagallen) [inline fairmlhealth/performance_metrics.py:3]: unused packages

**Review by camagallen — COMMENTED** [review]
LGTM

**Pull request merged**